chia/wallet/puzzles/clawback/drivers.py

```diff
@@ -179,3 +190,39 @@ def generate_clawback_spend_bundle(
         time_lock, metadata.sender_puzzle_hash, metadata.recipient_puzzle_hash, inner_puzzle, inner_solution
     )
     return CoinSpend(coin, puzzle, solution)
+
+
+async def add_clawback_outgoing_tx(
```

Comment:
Assumed we need to keep this because we really need both places to add it, is `chia.wallet.puzzles` really the right place for this?

Reply:
We don't have a wallet for the Clawback. The basic idea is to put this in the Clawback related files.